.env.example

```
@@ -10,13 +10,43 @@ NEXTAUTH_SECRET="your-secret-key-here-generate-with-openssl-rand-base64-32"
+# turn-credentials route signs short-lived HMAC credentials per session.
+# Unset = no TURN (voice falls back to STUN/host — LAN/local only).
+# TURN_URL="turn:turn.example.com:3478?transport=udp"
+# TURN_SECRET="your-coturn-static-auth-secret"
+# TURN_TTL=3600
+
 # Feature flags (redeploy to apply)
 FEATURE_VOICE_CHAT=true
 NEXT_PUBLIC_FEATURE_VOICE_CHAT=true
 FEATURE_AI_HINTS=false
-NEXT_PUBLIC_FEATURE_NEW_UI=false
+FEATURE_NEW_UI=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FEATURE_NEW_UI key should go before the FEATURE_VOICE_CHAT key

(UnorderedKey)